.travis.yml

```diff
@@ -4,31 +4,32 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 env:
-  - SYMFONY_VERSION="2.7.*,>2.7.6"
+  - SYMFONY_VERSION="3.0.*"
```

Comment:
without this the build fails with deprecation warnings as for some reason composer ends up picking very earlier Symfony versions